promptbook/models.py

```diff
@@ -41,6 +45,11 @@ class Prompt(models.Model):
     def __str__(self):
         return self.text[:50] + '...' if len(self.text) > 50 else self.text
 
+    # add unique together constraint for text, owner and category
+    class Meta:
+        unique_together = ['text', 'owner', 'category']
```

Comment:
If the constraint fails, Do we have an error message now on UI for user ?

Reply:
<img width="755" alt="Screenshot 2023-04-10 at 5 14 09 PM" src="https://user-images.githubusercontent.com/24906998/230930284-52f76684-fead-4b45-a09a-fe2c3c68cce2.png">
Yes, there is a message displayed on the UI for the user in case he tries to add the same brand in the same category which already exists.